wallet/txauthor/author_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 
 	"github.com/btcsuite/btcd/wire"
-	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcd/btcutil"
```

Comment:
`wallet/txauthor` has it's own go.mod to update this as well.  Will be needed after the neutrino update in https://github.com/lightninglabs/neutrino/pull/242 because neutrino does not require the main btcwallet module, just `wallet/txauthor` and `wxtmgr` (`walletdb`, too but it does not require btcutil)